docs/docs/providers/gcloud_iam.md

```diff
@@ -48,6 +64,13 @@ resources:
     policy:
       id: my_policy
       version: 1
+    roles:
```

Comment:
Can you add a section at the bottom explaining the way permissions are configured here similar to how it's done in the `bigquery` doc

Reply:
Done